samples/bluetooth/iso_time_sync/README.rst

```diff
@@ -0,0 +1,364 @@
+.. _bluetooth-isochronous-time-synchronization:
+
+Bluetooth: ISO time synchronization
+###################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+Overview
+********
+
+The Bluetooth® ISO time synchronization sample showcases time-synchronized processing of isochronous data.
+
+The sample demonstrates:
+
+ * How to send isochronous data on multiple streams in such a way that all receiving devices can process the data at the same time.
+ * How receiver(s) of the isochronous data can process received data synchronously.
+ * How to provide the isochronous data to the controller right before it is sent on air.
+ * How to achieve this for either broadcast (BIS) or over connections (CIS).
+
+This sample demonstrates this by configuring a single device as a transmitter of its **BTN1** state.
+The transmitting and receiving device(s) toggle **LED1** synchronously with microsecond accuracy.
+**LED2** is toggled upon sending or receiving the data to allow measurement of the minimal the end-to-end latency.
+
+Requirements
+************
+
+The sample supports the following development kits:
+
+.. table-from-sample-yaml::
+
+The sample also supports other development kits running SoftDevice Controller variants that support isochronous channels (see :ref:`SoftDevice Controller feature support <softdevice_controller>`).
```

Comment:
Is there a reason why we don't just add these kits to the table of supported kits?

Reply:
The data in the table is taken from the sample.yaml. Tipically that's maintained by the sample's maintainer. If dks running softdevice controller can build this sample, I don't see why they should not be listed there, if there are no other technical implications. In case of additional Kconfigs needed, these should be mentioned in the building and running section